doc/rtd/explanation/instancedata.rst

```diff
@@ -65,7 +65,6 @@ Using ``instance-data``
 
 ``instance-data`` can be used in:
 
-* :ref:`User data scripts<user_data_script>`.
```

Comment:
Why the removal? [It's a valid usage](https://cloudinit.readthedocs.io/en/latest/explanation/format.html#example-user-data-script)

Reply:
This was redundant, since user data scripts are documented on the user data formats page.

Reply:
Gotcha. As-is, it's a little weird though. The text says `Cloud-config` but links to the user data page (not your doing). We could update the text of the link to say 'user data' rather than 'cloud-config',  you can only use instance data in cloud-config and user scripts, so I'm not sure we should be saying it can be used with user data when there are caveats to that.

I think we should either specify cloud-config and user data scripts independently and be ok with the redundancy, or we should link to the jinja format and say any user data format supported by jinja.

Reply:
@TheRealFalcon good point. I just pushed an update, let me know what you think.